tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -837,6 +789,12 @@ class DatasetOpsTestBaseV2 : public DatasetOpsTestBase {
 
   Status CreateFactory(const DatasetParams& dataset_params,
                        FunctionDef* fdef) const;
+
+  // Copies the given tensors, storing them in the `inputs` vectors, and storing
+  // owned references to the copies in `created_tensors`.
+  void AddTensorInputs(const std::vector<Tensor>& tensors,
```

Comment:
consider inlining this since it's only used once and unlikely to be useful to subclasses

Reply:
Yeah, this function is inlined now.